### DISCUSSION TITLE
Option to hide the title on a resource form page

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [filamentphp/filament#1251] [Ideas]
I would really like to be able to hide the title on the resource form page as it takes a lot of space on the page and the breadcrumbs are enough for me in most cases. Is there a method or a property which I can set to `null` or something to hide this? If not, I would really like to make a PR with this functionality if you guys agree with an option to hide this title. 

<img width="496" alt="CleanShot 2022-01-16 at 11 51 28@2x" src="https://user-images.githubusercontent.com/6506510/149657120-6539d4cd-be9d-40cf-bf3d-26209fffd994.png">

**Comment by danharrin** [comment]
I don't mind hiding this, but where would you put this setting? On the page? In the config for every page?

**Reply by SebastiaanKloos** [reply]
I think we need an option to override the page title on a resource page. As you can see in my screenshot, it's not translating yet so that would also fix that issue. 

So adding an option to override (or adding `false` / `null` to hide the title) in `Resources/Pages/Page.php` would be the best option I guess. 

What do you think?

**Reply by danharrin** [reply]
The page title is already translated, can you double check that the key exists for your locale?

**Reply by SebastiaanKloos** [reply]
> The page title is already translated, can you double check that the key exists for your locale?

My bad🙈

**Comment by SebastiaanKloos** [comment]
I see the update context has a delete button on top on the right side of the title. So adding the option to remove the title, won't improve to the initial issue. I think I'm used to the trash icon button which October CMS uses ;-)
